functional-samples/cookbook.file_handlers/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "File Handling API",
+  "version": "1.0",
+  "description": "Shows how to use the file_handlers manifest key.",
+  "manifest_version": 3,
```

Comment:
Let's add the minimum_chrome_version here too

Reply:
Thanks, done! Sorry for repeatedly forgetting.